packages/eslint-plugin-lit-a11y/lib/rules/aria-attr-types.js

```diff
@@ -77,77 +93,58 @@ const validityCheck = (value, expectedType, permittedValues) => {
   }
 };
 
-module.exports = {
-  validityCheck,
+/** @type {import("eslint").Rule.RuleModule} */
+const AriaAttrTypesRule = {
   meta: {
+    type: 'suggestion',
     docs: {
       description: 'aria-attr-types',
-      category: 'Fill me in',
+      category: 'Accessibility',
       recommended: false,
     },
-    fixable: null, // or "code" or "whitespace"
-    schema: [
-      // fill in your schema
-    ],
+    fixable: null,
+    schema: [],
   },
 
   create(context) {
-    // variables should be defined here
-
-    //----------------------------------------------------------------------
-    // Helpers
-    //----------------------------------------------------------------------
-
-    // any helper functions should go here or else delete this section
-
-    //----------------------------------------------------------------------
-    // Public
-    //----------------------------------------------------------------------
-
     return {
-      TaggedTemplateExpression: node => {
-        if (
-          node.type === 'TaggedTemplateExpression' &&
-          node.tag.type === 'Identifier' &&
-          node.tag.name === 'html'
-        ) {
+      TaggedTemplateExpression(node) {
+        if (isHtmlTaggedTemplate(node)) {
           const analyzer = TemplateAnalyzer.create(node);
 
           analyzer.traverse({
             enterElement(element) {
-              const ariaAttributes = Object.keys(element.attribs).filter(attr =>
-                attr.startsWith('aria-'),
-              );
+              const ariaAttributes = Object.keys(element.attribs)
+                .map(attr => attr.toLowerCase())
```

Comment:
nitpick, but maybe the lowercasing could be done in `isAriaPropertyName` as well?

Or maybe we can even have a general `extractAriaAttributes(element.attribs)` helper function?

Reply:
I agree, I'll define that helper in `util/aria.js`